pkg/controllers/user/globaldns/user_cluster_ingress.go

```diff
@@ -116,59 +117,74 @@ func (ic *UserIngressController) noGlobalDNS() bool {
 	return len(gd) == 0
```

Comment:
`noGlobalDNS` has a possibility of returning error, in that case we don't return it back to framework, not sure if it was intended

Reply:
edited and returning error now